examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -270,6 +270,12 @@ def main():
         action="store_true",
         help="Enable deterministic generation using CPU Generator",
     )
+    parser.add_argument(
+        "--quant_mode",
+        default="disable",
+        type=str,
+        help="Quantization mode 'measure', 'quantize' or 'disable'",
+    )
```

Comment:
can you do the same as run_generation? and not introduce extra arguments?

Reply:
done